### PR TITLE
Map some fields related to stats and structures

### PR DESCRIPTION
**Pull request by NoahvdAa**

### CODE REVIEW

File: mappings/net/minecraft/stat/StatFormatter.mapping [hunk 1]
```diff
@@ -4,3 +4,13 @@ CLASS net/minecraft/class_3446 net/minecraft/stat/StatFormatter
 	FIELD field_16977 DISTANCE Lnet/minecraft/class_3446;
 	FIELD field_16978 DIVIDE_BY_TEN Lnet/minecraft/class_3446;
 	FIELD field_16979 TIME Lnet/minecraft/class_3446;
+	METHOD format format (I)Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/stat/StatFormatter.mapping]:
```suggestion
	METHOD format (I)Ljava/lang/String;
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/structure/DesertTempleGenerator.mapping [hunk 2]
```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_3346 net/minecraft/structure/DesertTempleGenerator
 	FIELD field_14397 hasPlacedChest [Z
+	FIELD field_34725 HEIGHT I
```
Comment (enbrain) [inline mappings/net/minecraft/structure/DesertTempleGenerator.mapping]: This seems to be the 5th (in one-based) argument for `ShiftableStructurePiece(StructurePieceType type, int x, int y, int z, int width, int height, int depth, Direction orientation)`.
```suggestion
	FIELD field_34725 WIDTH I
```

**Review by enbrain — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/structure/DesertTempleGenerator.mapping (hunk 2 above)
Comment (NoahvdAa) [reply mappings/net/minecraft/structure/DesertTempleGenerator.mapping]: Are you sure? It seems to be the fourth (or sixth, but that's already the depth)
```java
super(StructurePieceType.DESERT_TEMPLE, x, 64, z, 21, 15, 21, DesertTempleGenerator.getRandomHorizontalDirection(random));```

Comment (enbrain) [reply mappings/net/minecraft/structure/DesertTempleGenerator.mapping]: Yes. it's fourth in zero-based, but the fourth is width. You named it `HEIGHT`.

Comment (NoahvdAa) [reply mappings/net/minecraft/structure/DesertTempleGenerator.mapping]: My bad, I completely misread your comment 🤦.

**Review by enbrain — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**